lib/endpoints/class-wp-rest-posts-controller.php

```diff
@@ -1074,7 +1074,8 @@ protected function prepare_links( $post ) {
 			),
 		);
 
-		if ( in_array( $post->post_type, array( 'post', 'page' ) ) || post_type_supports( $post->post_type, 'author' ) ) {
+		if ( ( in_array( $post->post_type, array( 'post', 'page' ) ) || post_type_supports( $post->post_type, 'author' ) )
```

Comment:
To force the data to always be present, even if the developer has removed post type supports.

Our posts and pages are fixed schemas. Except not, of course.